docker/debian-9/Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM debian:9
 
 ARG chefver
 RUN apt-get update && apt-get install -y curl gnupg && \
-    curl -sL -o /tmp/chef.deb \
-        https://packages.chef.io/files/stable/chef/${chefver}/debian/8/chef_${chefver}-1_amd64.deb && \
+    curl -fsL -o /tmp/chef.deb \
+        https://packages.chef.io/files/stable/chef/${chefver}/debian/9/chef_${chefver}-1_amd64.deb && \
```

Comment:
somehow it was 8… (maybe, older Chef did not provide deb for Debian 9 and we used deb for Debian 8 instead)